rnd/autogpt_builder/src/components/CustomNode.tsx

```diff
@@ -297,7 +309,7 @@ const CustomNode: FC<NodeProps<CustomNodeData>> = ({ data, id }) => {
         return schema.enum ? (
           <div key={fullKey} className="input-container">
             <select
-              value={value || ''}
+              value={getSelectValue(value)}
```

Comment:
are there currently any cases where this is necessary?

Reply:
@aarushik93

Reply:
there's nothing in our nodes yet with different data types - but this fixes some of the TS check issues I was having - if there's a better solution lmk

Reply:
What was TypeScript's complaint?

Reply:
`Type error: Type '{ [key: string]: any; }' is not assignable to type 'string | number | readonly string[] | undefined'.`